indy_node/server/domain_req_handler.py

```diff
@@ -250,11 +270,38 @@ def _validate_claim_def(self, req: Request):
             schemaSeqNo=schema_ref,
             signatureType=signature_type
         )
+
+        try:
+            origin_role = self.idrCache.getRole(
+                req.identifier, isCommitted=False) or None
+        except BaseException:
+            raise UnknownIdentifier(
+                req.identifier,
+                req.reqId)
         if claim_def:
-            raise InvalidClientRequest(identifier, req.reqId,
-                                       '{} can have one and only one CLAIM_DEF for '
-                                       'and schema ref {} and signature type {}'
-                                       .format(identifier, schema_ref, signature_type))
+            origin = req.identifier
+            owner = self.idrCache.getOwnerFor(req.identifier, isCommitted=False)
```

Comment:
I think this check doesn't make sense. The owner is related to the DID who owns the corresponding verkey.
Here we just need to make sure that only the person who created the ClaimDef can modify it. But this is in fact always guaranteed since sender's DID (identifier) is part of the ClaimDef's key in the state trie.
So, I think that the whole check for whether we already have a Claim Def can be removed.